sql/src/main/java/org/vertexium/sql/SqlTableEdge.java

```diff
@@ -39,6 +41,7 @@ InMemoryTableEdge asInMemoryTableElement() {
 
     private static class InMemoryAdapter extends InMemoryTableEdge
             implements Storable<SqlTableElement<InMemoryEdge>, SqlGraph> {
+        private static final long serialVersionUID = SqlTableEdge.serialVersionUID;
```

Comment:
What's the reason for this `serialVersionUID` to be the same as its outer class?